### DISCUSSION TITLE
How to use a trained lora in the ui?

### DISCUSSION THREAD

**Discussion by dvelm** [lllyasviel/Fooocus#235] [Q&A]
I trained a Lora model and I want know how to use it in the web ui. Is it sufficient to load it in Lora section? I created my model with this colab https://youtu.be/f-GUSxCiYs8?si=NgA6mJY-zTTHxz8e . Any reccomendation? Loading the Lora I have created with the words I used doesn't give me the trained images

**Comment by barepixels** [comment]
Did you read the comments in YouTube?  Faulty tutorial, it only makes Lora for SD1.5

**Reply by dvelm** [reply]
Not true. I try it directly and I don't rely entirely on comments. The colab is training correctly the lora for SD XL and it gives the results when generating images from prompts on Colab. But in the Fooocus ui doesn't give the outputs correctly. The lora is loaded and the images are generated with lora but not too much. Increasing weight distorts the output

**Comment by MrQMarx** [comment]
I'm also wondering if it's loading my Lora. 
I used this one on full
https://civitai.com/models/112529/old-school-velma
and also a custom checkpoint 
I set the seed the same and the images are basically identical.
![2023-09-06_10-49-57_4732](https://github.com/lllyasviel/Fooocus/assets/22857189/ad8864fc-b35c-4b2a-97a8-ff0b049bd55a)
![2023-09-06_10-37-48_3059](https://github.com/lllyasviel/Fooocus/assets/22857189/6ad14357-42aa-4597-b632-1dd24926994b)

I checked the log.html and it says the settings are applied.
It just doesn't seem to make a difference?

It is my first time using LORA and checkpoints though so maybe it doesn't make as much difference as I thought?

**Reply by MrQMarx** [reply]
I have just found out that if you reference it in the prompt directly, you get better results.
`<lora:VelmaDogu:1>`
I am commenting, despite my embarrassment, in case this helps anyone including OP.

**Reply by mashb1t** [reply]
This is a 1.5 LoRA and is most likely only going to work correctly when using a 1.5 refiner.
![Screenshot 2024-01-30 at 21 44 09](https://github.com/lllyasviel/Fooocus/assets/9307310/6f14962c-e94a-45b5-9fdb-6d4b0ad1ec9f)

**Comment by gutsalovpavel** [comment]
Noticed, if to use 2 or 3 faceswap control net slots, with the different photos, plus Lora of the face, it gives a good results
